routes/api.js

```diff
@@ -2,19 +2,43 @@ const express = require('express');
 const CalendlyService = require('../services/calendlyService');
 const { isUserAuthenticated, formatEventDateTime } = require('../utils');
 const router = express.Router();
+const User = require('../models/userModel');
 
-router.get('/scheduled_events', isUserAuthenticated, async (req, res) => {
-    const { access_token, refresh_token } = req.user;
-    const { count, page_token, user } = req.query;
-    const calendlyService = new CalendlyService(access_token, refresh_token);
+router
+    .get('/scheduled_events', isUserAuthenticated, async (req, res) => {
+        const { access_token, refresh_token, calendly_uid } = req.user;
+        const { count, page_token } = req.query;
+        const calendlyService = new CalendlyService(access_token, refresh_token);
 
-    const {
-        collection,
-        pagination
-    } = await calendlyService.getUserScheduledEvents(user, count, page_token);
-    const events = collection.map(formatEventDateTime);
+        const {
+            collection,
+            pagination
+        } = await calendlyService.getUserScheduledEvents(calendly_uid, count, page_token);
+        const events = collection.map(formatEventDateTime);
```

Comment:
Looks as if there are some spacing issues! Hopefully, it's not too much of a pain to get rid of.

Reply:
I added/ran Prettier in the last two commits so we don't have to worry about this going forward.

Reply:
Beautiful! Thank you.